stage0/run.go

```diff
@@ -47,21 +43,17 @@ import (
 	"github.com/coreos/rocket/pkg/lock"
 	ptar "github.com/coreos/rocket/pkg/tar"
 	"github.com/coreos/rocket/version"
-
-	"github.com/coreos/rocket/stage0/stage1_init"
-	"github.com/coreos/rocket/stage0/stage1_rootfs"
 )
 
 const (
-	initPath  = "stage1/init"
+	initPath  = "stage1/rootfs/init"
```

Comment:
I was originally hoping/expecting that we would be able to use the image's app entrypoint to determine this at runtime instead of hardcoding it (https://github.com/coreos/rocket/issues/265#issuecomment-66814268)

But it's slightly more awkward since we also have the enter binary in there; I'm guessing that's why you just stuck to hardcoding this?